core/src/main/scala/kafka/server/KafkaConfig.scala

```diff
@@ -757,18 +757,14 @@ object KafkaConfig {
 
   }
 
-  def configNames() = {
-    import scala.collection.JavaConversions._
-    configDef.names().toList.sorted
-  }
+  def configNames() = configDef.names().asScala.toList.sorted
 
   /**
     * Check that property names are valid
     */
   def validateNames(props: Properties) {
-    import scala.collection.JavaConversions._
-    val names = configDef.names()
-    for (name <- props.keys)
+    val names = configDef.names().asScala
+    for (name <- props.keys.asScala.asInstanceOf[Set[String]])
```

Comment:
Unfortuantely props is a `Properties` == `Hashtable<Object, Object>` so the `props.keys` ends up as an `AnyRef`.